styleguide/derek/_partials/grid/widget/ias-no-hover.ejs

```diff
@@ -0,0 +1,18 @@
+<!-- Default description to allow for partial override to demonstrate really long text and equal height -->
+<% if(!desc) { var desc = "This description cannot be longer than 75 characters including all spaces."; } %>
+<!-- Infrastructure and Services Pattern -->
+
+<div class="ias">
+  <div class="row">
+    <div class="col-sm-4 col-xs-3 ias-img">
```

Comment:
@bighappyface do you think maybe the class here should be something else since it doesnt directly alter the img tag itself? Maybe something like ```ias-img-col``` or ```ias-img-container``` ?